release_util/management/commands/check_reserved_keywords.py

```diff
@@ -281,48 +305,76 @@ def set_status(violations, config):
         filter(lambda v: v not in config.overrides, violations)
     )
     if len(valid_violations) > 0:
-        click.secho("Found reserved keyword conflicts!", fg="red")
+        log.error("Found reserved keyword conflicts!")
         sys.exit(1)
```

Comment:
You don't really want to deviate from Django's standard pattern by exit()ing from handle().  Just throw a CommandError which is more likely to be forwards-compatible: https://docs.djangoproject.com/en/3.0/howto/custom-management-commands/#command-exceptions

Reply:
I was trying not to deviate. Perhaps you know of a way to handle this. I wanted to change some of the log lines to be debug, rather than info, as they weren't helpful in the CI workflow. However, I couldn't get them to display, even if I changed the `--verbosity` flag (https://docs.djangoproject.com/en/dev/ref/django-admin/#cmdoption-verbosity).

Reply:
Hence the mucking around with handlers.

Reply:
ah, oops, I see what you are saying. Will fix.

Reply:
To be fair, i'm the one that first migrated this code to a management command :P